modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIPublisherImpl.java

```diff
@@ -1687,4 +1694,159 @@ public HttpResponse deleteAPIProductRevision(String apiUUID, String revisionUUID
     public ApiResponse<APIKeyDTO> generateInternalApiKey(String apiId) throws ApiException {
         return apIsApi.generateInternalAPIKeyWithHttpInfo(apiId);
     }
+
+    /**
+     * Add comment to given API
+     *
+     * @param apiId   - api Id
+     * @param comment - comment to  add
+     * @param category - category of the comment
+     * @param replyTo - comment id of the root comment to add replies
+     * @return - http response of add comment
+     * @throws ApiException - throws if add comment fails
+     */
+
```

Comment:
Let's remove this unnecessary newline

Reply:
Fixed in https://github.com/wso2/product-apim/pull/13178